src/docker/api/Dockerfile.ws

```diff
@@ -1,13 +1,24 @@
 FROM debian:stable-slim
```

Comment:
Use https://hub.docker.com/_/haskell/, which will have the official stack toolchain already installed